pkg/querier/frontend2/querier_scheduler_worker.go

```diff
@@ -83,8 +83,8 @@ func NewQuerierSchedulerWorkers(cfg QuerierWorkersConfig, handler RequestHandler
 	}
 
 	frontendClientsGauge := promauto.With(reg).NewGauge(prometheus.GaugeOpts{
-		Name: "cortex_query_scheduler_worker_frontend_clients",
-		Help: "The current number of frontend clients.",
+		Name: "cortex_querier_query_frontend_clients",
```

Comment:
I would prefer it `cortex_query_scheduler` metrics prefix are reserved for the scheduler. Thoughts?

Reply:
:+1: Good idea, thanks for spotting.